crates/astria-sequencer/src/app.rs

```diff
@@ -1289,12 +1310,13 @@ mod test {
                 .unwrap(),
             value + 10u128.pow(19)
         );
+        let transfer_fee = app.state.get_transfer_base_fee().await.unwrap();
         assert_eq!(
             app.state
                 .get_account_balance(alice_address, native_asset)
                 .await
                 .unwrap(),
-            10u128.pow(19) - (value + TRANSFER_FEE),
+            10u128.pow(19) - (value + transfer_fee),
```

Comment:
We should really move this `tests` module into `src/app/tests.rs`. I wanted to comment several times now how I want saturating arithmetic. :-)